### ISSUE TITLE
export Exact type only once

### ISSUE THREAD

**Issue by xiaoyu-tamu** [dotansimha/graphql-code-generator#4226]
**Is your feature request related to a problem? Please describe.**
```yml
generates:
    preset: near-operation-file
    presetConfig:
      extension: .graphql.ts
      baseTypesPath: common/generated.ts
    plugins:
      - typescript-operations
      - typescript-react-apollo
```

I'm using near-operation-file plugin, After upgrading to 1.15.3, all generated files have 

`export type Exact<T extends { [key: string]: any }> = { [K in keyof T]: T[K] };`. 

I like to export all generated files in index.ts file, so other component can import it without nested import path.

```ts
// index.ts
export * from './a/a.generated.ts';
export * from './b/b.generated.ts';
```

But the problem is they all export type Exact.
```
Module './document.graphql' has already exported a member named 'Exact'. 
Consider explicitly re-exporting to resolve the ambiguity.ts(2308)
```

Can we reuse this type instead of define it in every generated files?

**Describe the solution you'd like**

Define Exact type near Maybe Type, other plugins use this like how they use Maybe.

```
// generated by near-operation-type plugin

import * as Types from 'some_base_file_base';

export type SomeQueryVariables = Types.Exact<{
  listingId: Types.Maybe<Types.Scalars['Int']>;
}>;

```
**Describe alternatives you've considered**

Do not export it.
```
`type Exact<T extends { [key: string]: any }> = { [K in keyof T]: T[K] };`. 

```

**Additional context**

**Comment by dotansimha** [comment]
You are right, @xiaoyu-tamu . Thank you for reporting this!

**Comment by dotansimha** [comment]
I guess the only way to avoid this, it to put the `Exact` type near `Maybe` (in `typescript` plugin). 

I've changed it in this PR: https://github.com/dotansimha/graphql-code-generator/pull/4235

@xiaoyu-tamu can you please try the alpha version `1.15.4-alpha-622ea4d0.0`?

**Comment by xiaoyu-tamu** [comment]
@xiaoyu-tamu Yes, it works

**Comment by dotansimha** [comment]
Fixed in v1.15.4
